Cpix/DrmSystem.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+using System.Xml.XPath;
+
+namespace Axinom.Cpix
+{
+	public sealed class DrmSystem : Entity
```

Comment:
This only looks like a limited set of fields. If we are supporting the DRMSystem element, we should support it in its entirety.

Reply:
Adding the other ones.